templates/bin/wso2server.sh.erb

```diff
@@ -315,7 +315,7 @@ do
     -Djava.net.preferIPv4Stack=true \
     -Dcom.ibm.cacheLocalHost=true \
     -DworkerNode=false \
-    -Dsetup \
```

Comment:
Shouldn't `-Dsetup` be included here for automatic deployments?

Reply:
-Dsetup is not included in the wso2server.sh